mflag: panic when printing usage with default FlagSet

In the `mflag` package, when using a default-initialized `FlagSet`, `flag.Parse` crashes whenever it attempts to print the usage. Usage writes directly on the FlagSet's output (which is nil when default-initialized) instead of using the .Out() facility (which defaults to os.Stderr if nil) ; writing on nil results in a panic.

The following code reproduces the problem:

```
package main

import (
        flag "github.com/docker/docker/pkg/mflag"
)

var (
        fDebug bool
        fSet   flag.FlagSet
)

func init() {
        fSet.BoolVar(&fDebug, []string{"D", "-debug"}, false, "Enable debug mode")
}

func main() {
        flag.Parse()
}
```

Results in the following trace:

```
➜  sandbox  ./sandbox -hsagasdg
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x20 pc=0x4447b6]

goroutine 1 [running]:
fmt.Fprintf(0x0, 0x0, 0x509190, 0xd, 0xc20802db20, 0x1, 0x1, 0x46b687, 0x0, 0x0)
        /home/mxs/go/src/fmt/print.go:189 +0xa6
github.com/docker/docker/pkg/mflag.func·001()
        /home/mxs/code/go/src/github.com/docker/docker/pkg/mflag/flag.go:563 +0x113
github.com/docker/docker/pkg/mflag.(*FlagSet).usage(0xc20805c000)
        /home/mxs/code/go/src/github.com/docker/docker/pkg/mflag/flag.go:881 +0x37
github.com/docker/docker/pkg/mflag.(*FlagSet).parseOne(0xc20805c000, 0xc2080322e0, 0x0, 0x0, 0x0, 0x0)
        /home/mxs/code/go/src/github.com/docker/docker/pkg/mflag/flag.go:955 +0x479
github.com/docker/docker/pkg/mflag.(*FlagSet).Parse(0xc20805c000, 0xc20800a010, 0x1, 0x1, 0x0, 0x0)
        /home/mxs/code/go/src/github.com/docker/docker/pkg/mflag/flag.go:1028 +0x45c
github.com/docker/docker/pkg/mflag.Parse()
        /home/mxs/code/go/src/github.com/docker/docker/pkg/mflag/flag.go:1101 +0x79
main.main()
        /home/mxs/code/go/src/sandbox/flag.go:17 +0x1b

goroutine 2 [runnable]:
runtime.forcegchelper()
        /home/mxs/go/src/runtime/proc.go:90
runtime.goexit()
        /home/mxs/go/src/runtime/asm_amd64.s:2232 +0x1

goroutine 3 [runnable]:
runtime.bgsweep()
        /home/mxs/go/src/runtime/mgc0.go:82
runtime.goexit()
        /home/mxs/go/src/runtime/asm_amd64.s:2232 +0x1

goroutine 4 [runnable]:
runtime.runfinq()
        /home/mxs/go/src/runtime/malloc.go:712
runtime.goexit()
        /home/mxs/go/src/runtime/asm_amd64.s:2232 +0x1
```

PR incoming.

--- a/pkg/mflag/flag.go
+++ b/pkg/mflag/flag.go
@@ -560,7 +560,7 @@ func defaultUsage(f *FlagSet) {
 // Usage prints to standard error a usage message documenting all defined command-line flags.
 // The function is a variable that may be changed to point to a custom function.
 var Usage = func() {
-	fmt.Fprintf(CommandLine.output, "Usage of %s:\n", os.Args[0])
+	fmt.Fprintf(CommandLine.Out(), "Usage of %s:\n", os.Args[0])
 	PrintDefaults()
 }
 

